content/en/docs/quickstart.md

```diff
@@ -14,15 +14,19 @@ Before you begin, you need:
 
 ## Create an OCI-compatible registry
 
-**WARNING:** The following example creates a registry with [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry). This registry should only be used for development purposes. When using other registries, ensure the registry is compatible with OCI Image specification v1.1.0. Starting with `v1.0.0-rc.1` of `notation`, by default, signatures are stored using [OCI Artifact Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md), which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)).
+**WARNING:** The following example creates a registry with [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry). This registry should only be used for development purposes. When using other registries, ensure the registry is compatible with OCI Image specification v1.1.0. Starting from Notation-v1.0.0-RC.3, signatures are stored using [OCI Image Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md) by default.
```

Comment:
Suggest updating "ensure the registry is compatible with OCI Image specification v1.1.0" to "ensure the registry is compatible with OCI specification v1.1.0", since there are two specs, one for OCI image spec, another is OCI distribution spec

Reply:
@yizha1 @toddysm Notation should be able to be compatible with OCI v1.1.0 and OCI v1.0.0 registries. I rephrased this sentence and link to [registries are compatible with Notary](https://notaryproject.dev/docs/faq/#what-registries-are-compatible-with-notary). It will be much more straightforward for users since people might not be aware of the OCI spec version that the registries support.